adm_program/languages/en.xml

```diff
@@ -1135,6 +1135,8 @@
   <string name="SYS_DELETED_USER">Deleted user</string>
   <string name="SYS_DESCRIPTION">Description</string>
   <string name="SYS_DESIGNATION">Name</string>
+  <string name="SYS_DIRECTORY_PROTECTION">Directory protection</string>
```

Comment:
Just for info: you must only change the de.xml . Thats the only one that will be uploaded to transifex. All other files will than be downloaded from transifex.